easybuild/easyblocks/m/motioncor2.py

```diff
@@ -56,6 +56,16 @@ def __init__(self, *args, **kwargs):
         if (LooseVersion(self.version) == LooseVersion("1.3.1")):
             self.motioncor2_verstring = "v%s" % self.version
 
+    @staticmethod
+    def extra_options(extra_vars=None):
+        extra = {
+            'skip_rpath_check': [False, "Skip rpath check", None],
```

Comment:
I would clarify the help string a bit here, and maybe also rename to `skip_rpath_sanity_check`

```python
"Skip RPATH sanity check"
```

@stefan-wolfsheimer Can you clarify why you need this option, in which context?